config/checkstyle/checkstyle.xml

```diff
@@ -11,6 +11,11 @@
         <property name="files" value="^.*/build/generated/.*\.java$"/>
     </module>
 
+    <module name="SuppressionSingleFilter">
```

Comment:
Pls add a comment that it's needed specifically for Windows because Windows, unlike Unix-like systems (Linux, MacOS etc.) uses `\` and not `/` to separate path segments